### PR TITLE
Implemented SectionControl.

### PR DESCRIPTION
**Pull request by grokys**
We were previously using the one supplied by TeamFoundation but we shouldn't be taking a dependency on TeamFoundation controls, as they're not part of the API.

### CODE REVIEW

File: src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml [hunk 1]
```diff
@@ -293,50 +293,61 @@
...
+                                                        <Setter Property="Header" Value="{x:Static prop:Resources.SwitchToListView}"/>
+                                                    </DataTrigger>
+                                                    <DataTrigger Binding="{Binding ChangedFilesViewType}" Value="ListView">
+                                                        <Setter Property="Header" Value="{x:Static prop:Resources.SwitchToTreeView}"/>
+                                                    </DataTrigger>
+                                                </Style.Triggers>
+                                            </Style>
+                                        </MenuItem.Style>
+                                        <MenuItem.Icon>
+                                            <ui:OcticonPath Icon="list_unordered" Fill="Black" Height="16"/>
```
Comment (shana) [inline src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml]: Is the fill always black here, is this going to work on theme changes?

Comment (grokys) [inline src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml]: Ah yes good point!

Comment (grokys) [inline src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml]: Oh no, false alarm: this icon is in a popup menu, and they don't change color according to theme.

Comment (shana) [inline src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml]: Ah cool! I don't think I saw any other thingies that should be themeable or defined somewhere else, so all good!

**Comment by shana** [comment]
bb8_thumbs_up

**Pull request merged**